pinot-broker/src/test/java/org/apache/pinot/broker/queryquota/HelixExternalViewBasedQueryQuotaManagerTest.java

```diff
@@ -149,7 +149,7 @@ public void testOfflineTableNotnullQuota()
     ZKMetadataProvider
         .setOfflineTableConfig(_testPropertyStore, OFFLINE_TABLE_NAME, TableConfigUtils.toZNRecord(tableConfig));
     setQps(tableConfig);
-    _queryQuotaManager.initTableQueryQuota(tableConfig, brokerResource);
+    _queryQuotaManager.initOrUpdateTableQueryQuota(tableConfig, brokerResource);
```

Comment:
Can we add the test case that's covering the following case?

1. Create ratelimiter
2. run some acquire
3. Check brokerMetrics table gauge value
4. Change the ratelimiter config
4. run some acquire
5. Check brokerMetrics table gauge value

Reply:
Discussed offline. It's a bit hard to add the test since the value callback function cannot be directly fetched from broker metrics.